ktlint-cli/build.gradle.kts

```diff
@@ -74,13 +74,17 @@ val shadowJarExecutable by tasks.registering(DefaultTask::class) {
 
         execFile.appendBytes(inputs.files.singleFile.readBytes())
         execFile.setExecutable(true, false)
-        if (!version.toString().endsWith("SNAPSHOT")) {
-            signing.sign(execFile)
-        }
     }
     finalizedBy(tasks.named("shadowJarExecutableChecksum"))
 }
 
+tasks.signMavenPublication {
+    dependsOn(shadowJarExecutable)
+    if (!version.toString().endsWith("SNAPSHOT")) {
+        sign(*shadowJarExecutable.map { it.outputs.files.files }.get().toTypedArray())
+    }
+}
```

Comment:
- Before: signed `execFile` only (exclude asc file), this might be intentd?
- Now: signing all files under `execFile` path.